test/test-options.jl

```diff
@@ -17,6 +17,47 @@
     )
 end
 
+@testset "Test run_scenario arguments" begin
+    connection = DBInterface.connect(DuckDB.DB)
+    _read_csv_folder(connection, joinpath(INPUT_FOLDER, "Tiny"))
```

Comment:
```suggestion
    connection = _tiny_fixture()
```
This is new-ish, we are only using it in the test-data-validation.jl at the moment